frontend/src/lib/components/Input.svelte

```diff
@@ -1,8 +1,10 @@
 <script lang="ts">
+  import ErrorMessage from '$lib/components/ErrorMessage.svelte';
+
   export let inputId: string;
   export let labelText: string;
   export let placeholder: string;
-  export let errorMessage: string;
+  export let errorMessage: string | null = null;
```

Comment:
This makes the error message optional as discussed. Now for an error to show `isError` has to be `true` and `errorMessage` has to be set. in cases where a field doesn't need an error `errorMessage` can be skipped now.